controllers/etcd_controller_test.go

```diff
@@ -1004,9 +977,6 @@ func validateEtcd(s *appsv1.StatefulSet, cm *corev1.ConfigMap, svc *corev1.Servi
 				}),
 			}),
 		}),
-		"Data": MatchKeys(IgnoreExtras, Keys{
-			"bootstrap.sh": Equal(fmt.Sprintf(bootstrapScriptWithTLS, instance.Name, backupPort)),
-		}),
```

Comment:
Any idea why there wasn't a check for config file here? Should we add it?

Reply:
We do have tests for etcd config file [here](https://github.com/gardener/etcd-druid/blob/ba48ea629e59585d1b71060c9fd91c8778a60ad6/controllers/etcd_controller_test.go#L660) (without TLS) and [here](https://github.com/gardener/etcd-druid/blob/ba48ea629e59585d1b71060c9fd91c8778a60ad6/controllers/etcd_controller_test.go#L1012) (with TLS)